ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/PlatformCve/PlatformCvePage.tsx

```diff
@@ -81,10 +79,67 @@ function PlatformCvePage() {
             </PageSection>
             <Divider component="div" />
             <PageSection variant="light">
-                <CvePageHeader data={platformCveMetadata} />
+                <CvePageHeader data={metadataRequest.data?.platformCVE} />
             </PageSection>
             <Divider component="div" />
             <PageSection className="pf-v5-u-flex-grow-1">
+                <div className="pf-v5-u-background-color-100 pf-v5-u-p-lg">
+                    {metadataRequest.error && (
+                        <Alert
+                            title="There was an error loading the summary data for this deployment"
+                            isInline
+                            variant="danger"
+                        >
+                            {getAxiosErrorMessage(metadataRequest.error)}
+                        </Alert>
+                    )}
+                    {metadataRequest.loading && (
+                        <Gallery
+                            hasGutter
+                            minWidths={{
+                                // Enforce a 1/3 size, taking into account the GridGap
+                                default: 'calc(33.3% - var(--pf-v5-l-gallery--m-gutter--GridGap))',
+                            }}
+                        >
```

Comment:
In a future follow up, I may extract this `<Gallery>` to a custom component to use as the "SummaryCardLayout" throughout VM 2.0, ideally with the Skeleton case abstracted as well to reduce the duplication.

Reply:
> ideally with the Skeleton case abstracted as well to reduce the duplication

Aha, a lite application of `TbodyUnified` pattern ;)

Reply:
I'm considering it. I am trying to avoid seeing everything as a nail though now that we have this new hammer. 😄